docs/Assets/AddingAnimationsFromUnity.md

```diff
@@ -3,12 +3,76 @@ id: AddingAnimationsFromUnity
 title: Adding Animations from Unity
 ---
 
-## Coming Soon!
+Assets that have animations can be used with Enklu. The platform supports animations that play on their own or are driven through scripting. This guide will teach you how to configure an Animator Controller in Unity to be used in your experiences. It's recommended to have a basic familiarity with Unity, and to have already read the Import Your Own Assets guide. If you'd like to follow along with this guide, you can get the assets used here.
```

Comment:
Need to add Google Drive link here.

Reply:
We have the option of storing the files in git _if we wanted_ so we don't need an external drive link. Though I'm also not a huge fan of throwing tons of zip files in git and making the clone/deploy take even longer 😓

Reply:
Yeah, I as thinking about that but cringed add committing zip files. I set up a CDN awhile back for web assets. We could use it for this sort of thing too.